csss-site/src/events/frosh/urls.py

```diff
@@ -45,5 +45,9 @@
     url(r'^2022$', frosh2022.index, name="Frosh Week 2022 Entrance"),
     url(r'^2022/frosh$', frosh2022.frosh, name="Frosh Week 2022"),
     url(r'^2022/secret$', frosh2022.secret, name="Not A Secret"),
+    # url(r'^2023$', frosh2023.index, name="Frosh Week 2023 Entrance"),
```

Comment:
i don't like this approach you have taken, i'd rather this be un-commented but instead redirect to a temporary page we already use for such occasions. any issues with me making the change myself?

Reply:
sure, go for it!